server/area_manager.py

```diff
@@ -751,7 +758,7 @@ def load_areas(self):
                     item['area'])
             self.areas.append(
                 self.Area(self.cur_id, self.server, item['area'],
-                          item['background'], item['bglock'],
+                          item['background'], item['bglock'], item['custom_bg_allowed'],
                           item['evidence_mod'], item['locking_allowed'],
                           item['iniswap_allowed'],
                           item['showname_changes_allowed'],
```

Comment:
Actually now that this was changed, I think you can actually get away with doing
```python
self.Area(self.cur_id, self.server, **item)
```
and then the default parameters set in `item` just above become redundant.